nbdev_nbs/statistics/regression.ipynb

```diff
@@ -63,12 +63,12 @@
eIiIj4lIIPERER8amAq3Bq6XOXnp7u55GIiIiIqyzXbVf61QZc8HHq1CkAqlSp4ueRiIiIiLtOnTpFbGysw2MMJldCFB8yGo0cOnSI0qVLYzB4tiFYeno6VapU4cCBA07b/RYFer1FW3F7vVD8XrNeb9FW1F6vyWTi1KlTJCYmEhLiOKsj4GY+QkJCqFy5slefIyYmpkj8ol2l11u0FbfXC8XvNev1Fm1F6fU6m/GwUMKpiIiI+JSCDxEREfGpYhV8RERE8MwzzxAREeHvofiEXm/RVtxeLxS/16zXW7QVt9ebV8AlnIqIiEjRVqxmPkRERMT/FHyIiIiITyn4EBEREZ9S8CEiIiI+VWyCj7fffpsaNWoQGRlJy5Yt+eWXX/w9JK+ZPHkyrVu3pnTp0lSoUIHrrruOv/76y9/D8pnJkydjMBgYPXq0v4fiNQcPHmTIkCGUK1eOkiVL0qxZMzZt2uTvYXlFdnY2Tz31FDVq1CAqKoqaNWvy7LPPYjQa/T00j1i5ciXXXnstiYmJGAwGvvvuO6v7TSYT48ePJzExkaioKDp37sz27dv9M1gPcPR6z58/z9ixY2ncuDHR0dEkJiZy6623cujQIf8N2AOc/Y7zuvvuuzEYDEydOtVn4/OHYhF8fPHFF4wePZonn3ySLVu2cMUVV9CrVy/279/v76F5xYoVKxg5ciTr1q1j0aJFZGdn06NHDzIyMvw9NK/bsGEDM2bMoEmTJv4eitecOHGCDh06EBYWxk8//cSOHTt45ZVXKFOmjL+H5hVTpkzhnXfeYdq0aezcuZMXX3yRl156iTfffNPfQ/OIjIwMmjZtyrRp02ze/+KLL/Lqq68ybdo0NmzYQHx8PN27d8/tgxVsHL3eM2fOsHnzZsaNG8fmzZv55ptv+Pvvv+nbt68fRuo5zn7HFt999x2//voriYmJPhqZH5mKgTZt2pjuueceq9vq169veuyxx/w0It86fPiwCTCtWLHC30PxqlOnTpnq1KljWrRokalTp06mBx54wN9D8oqxY8eaOnbs6O9h+Ezv3r1Nt99+u9Vt119/vWnIkCF+GpH3AKZvv/0293uj0WiKj483vfDCC7m3ZWZmmmJjY03vvPOOH0boWflfry3r1683AaZ9+/b5ZlBeZu81//vvv6ZKlSqZtm3bZqpWrZrptdde8/nYfKnIz3ycO3eOTZs20aNHD6vbe/TowZo1a/w0Kt9KS0sDIC4uzs8j8a6RI0fSu3dvunXr5u+heNW8efNo1aoVN954IxUqVKB58+a89957/h6W13Ts2JElS5bw999/A/Dbb7+xatUqrrnmGj+PzPv27NlDamqq1ftXREQEnTp1KlbvXwaDocjO7IG5oerQoUN55JFHaNSokb+H4xMB11jO044ePUpOTg4VK1a0ur1ixYqkpqb6aVS+YzKZGDNmDB07diQpKcnfw/Gazz//nM2bN7NhwwZ/D8Xr/vnnH6ZPn86YMWN44oknWL9+Pffffz8RERHceuut/h6ex40dO5a0tDTq169PaGgoOTk5PP/88wwaNMjfQ/M6y3uUrfevffv2+WNIPpWZmcljjz3GLbfcUmQar9kyZcoUSpQowf333+/vofhMkQ8+LAwGg9X3JpOpwG1F0ahRo/j9999ZtWqVv4fiNQcOHOCBBx5g4cKFREZG+ns4Xmc0GmnVqhWTJk0CoHnz5mzfvp3p06cXyeDjiy++YNasWcyePZtGjRqxdetWRo8eTWJiIsOGDfP38HyiOL5/nT9/noEDB2I0Gnn77bf9PRyv2bRpE6+//jqbN28u8r/TvIr8skv58uUJDQ0tMMtx+PDhAp8mipr77ruPefPmsWzZMipXruzv4XjNpk2bOHz4MC1btqREiRKUKFGCFStW8MYbb1CiRAlycnL8PUSPSkhIoGHDhla3NWjQoMgmUD/yyCM89thjDBw4kMaNGzN06FAefPBBJk+e7O+heV18fDxAsXv/On/+PDfddBN79uxh0aJFRXrW45dffuHw4cNUrVo19/1r3759PPTQQ1SvXt3fw/OaIh98hIeH07JlSxYtWmR1+6JFi2jfvr2fRuVdJpOJUaNG8c0337B06VJq1Kjh7yF5VdeuXfnjjz/YunVr7p9WrVoxePBgtm7dSmhoqL+H6FEdOnQosHX677//plq1an4akXedOXOGkBDrt6rQ0NAis9XWkRo1ahAfH2/1/nXu3DlWrFhRZN+/LIHHrl27WLx4MeXKlfP3kLxq6NCh/P7771bvX4mJiTzyyCP8/PPP/h6e1xSLZZcxY8YwdOhQWrVqRbt27ZgxYwb79+/nnnvu8ffQvGLkyJHMnj2buXPnUrp06dxPTbGxsURFRfl5dJ5XunTpAvks0dHRlCtXrkjmuTz44IO0b9+eSZMmcdNNN7F+/XpmzJjBjBkz/D00r7j22mt5/vnnqVq1Ko0aNWLLli28+uqr3H777f4emkecPn2a3bt3536/Z88etm7dSlxcHFWrVmX06NFMmjSJOnXqUKdOHSZNmkTJkiW55ZZb/DjqwnP0ehMTExkwYACbN2/m+++/JycnJ/f9Ky4ujvDwcH8N+5I4+x3nD7DCwsKIj4+nXr16vh6q7/h3s43vvPXWW6Zq1aqZwsPDTS1atCjS204Bm39mzpzp76H5TFHeamsymUzz5883JSUlmSIiIkz169c3zZgxw99D8pr09HTTAw88YKpataopMjLSVLNmTdOTTz5pysrK8vfQPGLZsmU2/78OGzbMZDKZt9s+88wzpvj4eFNERITpyiuvNP3xxx/+HfQlcPR69+zZY/f9a9myZf4eeqE5+x3nVxy22hpMJpPJR3GOiIiISNHP+RAREZHAouBDREREfErBh4iIiPiUgg8RERHxKQUfIiIi4lMKPkRERMSnFHyIiIiITyn4EBEREZ9S8CEiIiI+peBDREREfErBh4iIiPiUgg8RERHxqf8HVXJA4a9gUIwAAAAASUVORK5CYII=",
```

Comment:
I think it is better to use a fixed random seed to generate the data points, otherwise images are always changed for each push, which might make this repo "unexpectedly" larger